ShapeVariationAnalyzer/ShapeVariationAnalyzer.py

```diff
@@ -2246,8 +2246,23 @@ def addColorMap(self, table, dictVTKFiles):
                     polyData = append.GetOutput()
 
                     vtkFile = os.path.splitext(vtkFile)[0]+'.vtk'
+                elif os.path.basename(vtkFile).endswith(".srep.json"):
+                    storage = slicer.mrmlScene.AddNewNodeByClass('vtkMRMLSRepStorageNode')
+                    storage.SetFileName(vtkFile)
+                    try:
+                        srep = storage.CreateSRepNode("name")
+                    except:
```

Comment:
Is there a more specific exception that could be intercepted here ?

Reply:
Will update to not return False, but rather re-raise. In that case, I think the most generic `except` is appropriate since the desired action is "cleanup storage node if we aren't setting it as an srep's storage node"

Reply:
Updated per my previous comment.